### PR TITLE
Add version bump warning

### PR DESCRIPTION
**Pull request by or-segal**
Warn about a missing version bump for a package.
image

### CODE REVIEW

**Review by raphaelboukara — APPROVED** [review]
🍾

File: src/versionBumpWarn/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * @returns {Promise<undefined>}
+ */
+const run = async(fileContents, diffForFile, exist, warn) => {
+    const condition = await exist('package.json');
+    if (!condition) {
+    // Required files do not exist
+        return;
+    }
+
+    const packageJsonContent = await fileContents('package.json');
```
Comment (raphaelboukara) [inline src/versionBumpWarn/index.js:25]: what will happen in case there is no `package.json`? ex: a ruby based repos

**Review by raphaelboukara — CHANGES_REQUESTED** [review]

File: src/versionBumpWarn/index.js (hunk 1 above)
Comment (raphaelboukara) [reply src/versionBumpWarn/index.js:25]: if there is no package.json, just return without warnings

Comment (or-segal) [reply src/versionBumpWarn/index.js:25]: that's what happens (line 20)

**Review by raphaelboukara — APPROVED** [review]

**Pull request merged**